data-prep-pipeline/data_prep_pipeline/update_hgnc_gene_table.conseq

```diff
@@ -0,0 +1,18 @@
+rule update_hgnc_gene_table:
+    inputs:
+        hgnc_gene_table={"type": "hgnc_gene_table"}
+    outputs:
+        {
+            "type": "gene",
+            "filename": {"$filename": "hgnc_gene_table.csv"},
+            "orig_dataset_id": "{{inputs.hgnc_gene_table.dataset_id}}"
+        }
+    run "python3" with """
+        from taigapy import create_taiga_client_v3
+
+        tc = create_taiga_client_v3()
+        print("Getting HGNC gene data...")
+        hgnc_gene_df = tc.get("{{inputs.hgnc_gene_table.dataset_id}}")
```

Comment:
Instead of downloading the CSV and uploading a new CSV, can you add the file to destination with a `TaigaReference`? That way taiga will track the provenance of the original file, and we won't end up with two taiga datasets which are identical.